cmd/podman/libpodruntime/runtime.go

```diff
@@ -67,7 +67,10 @@ func getRuntime(ctx context.Context, c *cliconfig.PodmanCommand, renumber bool,
 	if c.Flags().Changed("storage-driver") {
 		storageSet = true
 		storageOpts.GraphDriverName = c.GlobalFlags.StorageDriver
+		// Overriding the default storage driver caused GraphDriverOptions from storage.conf to be ignored
+		storageOpts.GraphDriverOptions = []string{}
```

Comment:
Does this actually fix it? I think it's Mount options that are the problem, not graph driver options

Reply:
Ok that might be true.  But this is definitely a bug that should be fixed.

Reply:
After looking at the code.  mountopt is a storageopt, so this should fix the issue.